lib/store/games.js

```diff
@@ -32,6 +32,18 @@ const GameModel = {
       return rootGetters.findAllComponents(game.componentIds)
     },
 
+    findAllPrintableGameComponents: (_, __, ___, rootGetters) => game => {
+      return rootGetters.findAllComponents(game.componentIds).filter(c => {
```

Comment:
Here's something I try to stick to: full word variable names, no abbreviations or initials. They just get impossible to work with as the codebase grows, so I take the hit early to keep my code overly readable.

Reply:
Another thing here is the use of `findAll` and then a `filter`, I'm thinking that we've already got some power in a `search*` method or something from my CRUD resource scaffolding. I'm going to poke around and try to find a tighter way to do this getter...